Remove Ansible at this phase

kyliel opened the pull request:
Remove Ansible at this phase. We will add it back around Jan/Feb with official announcement.

azurecla commented:
Hi __@normalian__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

deneha commented:
@normalian Thank you for this contribution.  I will have the author review.

@yidingzhou  Could you please review and approve/disapprove this change?

YidingZhou commented:
signed off.

kyliel committed:
Remove Ansible at this phase 

Remove Ansible at this phase. We will add it back around Jan/Feb with official announcement.

kyliel committed:
update the format

kyliel committed:
update format
